src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -279,6 +280,10 @@ export default class StyleConfigurator {
         // align settings
         styleValues.set(mxConstants.STYLE_LABEL_POSITION, mxConstants.ALIGN_TOP);
         styleValues.set(mxConstants.STYLE_VERTICAL_LABEL_POSITION, mxConstants.ALIGN_LEFT);
+        // TODO keep default in that case + add test even if this modification is not accepted
+        if (bpmnCell.bpmnElement.kind == ShapeBpmnElementKind.TEXT_ANNOTATION) {
+          styleValues.set(mxConstants.STYLE_ALIGN, mxConstants.ALIGN_LEFT);
```

Comment:
We don't need that if it already added in the style.

Reply:
I think this one has been added from @tbouffard suggestion
Is there some specific situation where we need the alignment here?

Reply:
To keep concistency, as we didn't have this for the other styles, I would not have put anything here.

Reply:
I have just changed this in e791e3c  (the TODO was a reminder)

Reply:
I saw the modification. LGTM